ground/src/main/java/com/google/android/ground/ui/datacollection/tasks/AbstractTaskFragment.kt

```diff
@@ -179,6 +186,10 @@ abstract class AbstractTaskFragment<T : AbstractTaskViewModel> : AbstractFragmen
     lifecycleScope.launch { dataCollectionViewModel.onNextClicked(viewModel) }
   }
 
+  private fun saveCurrentState() {
+    dataCollectionViewModel.saveCurrentState(viewModel)
+  }
+
```

Comment:
Same as previous comment

Reply:
can we access `viewModel ` which is a AbstractTaskViewModel inside the DatCollectionfragment?

Reply:
DataCollectionViewModel can directly get the currently visible task's view model. We don't need to pass it to the `DataCollectionFragment`